doc/tools/docgram/common.edit_mlg

```diff
@@ -1211,11 +1202,11 @@ command: [
 | REPLACE "Next" "Obligation" "of" ident withtac
 | WITH "Next" "Obligation" OPT ( "of" ident ) withtac
 | DELETE "Next" "Obligation" withtac
-| REPLACE "Obligation" int "of" ident ":" lglob withtac
-| WITH "Obligation" int OPT ( "of" ident ) OPT ( ":" lglob withtac )
-| DELETE "Obligation" int "of" ident withtac
-| DELETE "Obligation" int ":" lglob withtac
-| DELETE "Obligation" int withtac
+| REPLACE "Obligation" integer "of" ident ":" lglob withtac
+| WITH "Obligation" integer OPT ( "of" ident ) OPT ( ":" lglob withtac )
+| DELETE "Obligation" integer "of" ident withtac
+| DELETE "Obligation" integer ":" lglob withtac
+| DELETE "Obligation" integer withtac
```

Comment:
Does an `integer` really make sense here and below (as well as in `g_obligations.mlg`)? The doc of `Program` above says `natural`

Reply:
Indeed https://github.com/coq/coq/blob/b972cc5195e941633319c1fa428a9801ac4ef9e2/vernac/declare.ml#L2272-L2277